design-editor/src/events-emitter.js

```diff
@@ -95,7 +95,9 @@ const EVENTS = {
 	PopupOpened : 'global:Popup.opened',
 	ChangeCapability: 'attributes:change-capability',
 	ChangeCapabilityParam: 'attributes:change-capability-params',
-	TAULoaded: 'editor:TAU-loaded'
+	TAULoaded: 'editor:TAU-loaded',
+	Undo: 'editor-undo',
```

Comment:
all events are composed from two parts connected with ":"
could ou keep the naming convention?

Reply:
Ok done.